packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/update/output.js

```diff
@@ -0,0 +1,26 @@
+var Foo =
+/*#__PURE__*/
+function () {
+  "use strict";
+
+  function Foo() {
+    babelHelpers.classCallCheck(this, Foo);
+    Object.defineProperty(this, _foo, {
+      writable: true,
+      value: 0
+    });
+  }
+
+  babelHelpers.createClass(Foo, [{
+    key: "test",
+    value: function test(other) {
+      babelHelpers.classPrivateFieldLooseBase(this, _foo)[_foo]++;
+      ++babelHelpers.classPrivateFieldLooseBase(this, _foo)[_foo];
+      babelHelpers.classPrivateFieldLooseBase(other.obj, _foo)[_foo]++;
+      ++babelHelpers.classPrivateFieldLooseBase(other.obj, _foo)[_foo];
+    }
+  }]);
+  return Foo;
+}();
+
+var _foo = babelHelpers.classPrivateFieldLooseKey("foo");
```

Comment:
Can we move these things inside the class closure? IIRC, @Andarist implemented something like `@babel/helper-get-class-iife` to do that.

Reply:
Oh it has not been merged yet - #6963